cli/types/cypress.d.ts

```diff
@@ -54,14 +57,11 @@ declare namespace Cypress {
   }
 
   interface RemoteState {
```

Comment:
Type here was originally removed, but then decorated. A `TODO` item here likely needs to be to migrate this type to the `internal-types.d.ts` file to satisfy [this conversation](https://github.com/cypress-io/cypress/pull/18075/files#r852327440)

Reply:
Or maybe the `internal-types-lite.d.ts` that was recently added?